src/transport_ofi.h

```diff
@@ -1353,8 +1354,8 @@ void shmem_transport_atomic_fetch(shmem_transport_ctx_t* ctx, void *target,
     shmem_transport_fetch_atomic_nbi(ctx, (void *) source, (const void *) &dummy,
                                      target, len, pe, FI_SUM, datatype);
 #else
-    shmem_transport_fetch_atomic(ctx, (void *) source, (const void *) NULL,
-                                 target, len, pe, FI_ATOMIC_READ, datatype);
+    shmem_transport_fetch_atomic_nbi(ctx, (void *) source, (const void *) NULL,
+                                     target, len, pe, FI_ATOMIC_READ, datatype);
```

Comment:
Is this change related to the MR descriptors?  I'm not sure I understand the reason for the change, can you please explain?  It seems relatively harmless... but wouldn't `shmem_transport_atomic_swap` need this change also (for consistency)?

Reply:
No, this is not related. But, the reason of this change is after this call, there is always a `get_wait` call which ensures completion in the internal layer. I might have missed `swap`, let me take a look.